.github/workflows/boards_build.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: Build supported boards
+on:
+  push:
```

Comment:
With this, the build will run on every single commit. Hence the path protection.

Reply:
Yes, see my previous comment. I think it's premature optimization to do this in this pull-request.

> Removed the path rules in the new workflow, to keep this pull-request focused on removing the patch.
> 
> I don't think the `build_boards` is slow enough to justify specialized path rules, but if so we can do that in a separate pull-request.

Reply:
Done.